meeshkan/server/server/response_matcher.py

```diff
@@ -56,35 +65,51 @@ def _exact_match(self, recording, real):
         if recording['pathname'] != real['pathname']:
             return False
 
-        if recording['query'] != real['query']:
-            return False
+        if 'query' in recording:
```

Comment:
Because of the poor python type checking, it's impossible to guarantee by here that it will actually be lowercase. This is one of the reasons I think that, in general, Python struggles as a language for type-heavy workloads like ours.  It requires lots of runtime checks and redundancy :(